onager/backends/slurm.py

```diff
@@ -60,7 +92,12 @@ def get_job_list(self, args):
         # jobid with subprocess.check_output()
         base_cmd += '--parsable '
 
-        base_cmd += "--array={}".format(args.tasklist)
+        num_jobs = len(args.tasklist.split(",")) / args.tasks_per_job
```

Comment:
The job IDs that you pass to slurm are no longer what you actually pass in. Instead, it's more of an index. For example, if tasklist is 2,3,5,7,11,13 and tasks_per_job is 2, then job ID of 2 refers to task 5 and 7.